axual-webclient/applications.go

```diff
@@ -53,7 +53,7 @@ func (c *Client) CreateApplication(data ApplicationRequest) (*ApplicationRespons
 }
 func (c *Client) GetApplicationByName(name string) (*ApplicationByNameResponse, error) {
```

Comment:
This is not correct name for the method. GetApplicationByAttributes would be correct